onnx/test/version_converter_test.py

```diff
@@ -1984,6 +1985,64 @@ def test_softmax_12_13(self) -> None:
         assert converted_model.graph.node[3].op_type == "Reshape"
         assert converted_model.opset_import[0].version == 13
 
+    @parameterized.parameterized.expand(
+        [
+            ((16, 3), (1,), True),  # per-tensor - compatible
```

Comment:
The first parameter of the test method can be conventionally set as `_: str`. Then I would supply this string as the first argument. This way they show up as test names, making it easy to read when any tests fail.

Reply:
That's indeed a neat solution. Thanks.